release-notes-v3.0.md

```diff
@@ -0,0 +1,109 @@
+Intel® Neural Compressor v3.0 Release
+
+- Highlights
+- Features
+- Improvements
+- Examples
+- Bug Fixes
+- Documentations
+- External Contributions
+- Validated Configurations
+
+**Highlights**
+ - FP8 quantization and INT4 model loading support on [Intel® Gaudi® AI accelerator](https://habana.ai/products/gaudi2/) 
+ - Framework extension API (3.x API) for quantization, mixed-precision and benchmarking  
```

Comment:
Do we need to mention 3.x here?

Reply:
I found "3.x" was used a lot in PR and also example file path, so I added this to indicate it is the same thing as framework extension API.  For release notes, I am ok to remove it if we believe it is kind of informal.

Reply:
removed